src/planscape/forsys/parse_forsys_output.py

```diff
@@ -86,11 +86,11 @@ class ForsysRankingOutputForMultipleScenarios():
     # Of note, priorities must be listed in the same format and order they're
     # listed for the forsys call.
     def __init__(
-            self, raw_forsys_output: "rpy2.robjects.vectors.ListVector",
+            self, raw_forsys_output: dict,
```

Comment:
I like the idea of moving the parsing step outside of this class!

nit: can "dict" be slightly more specific? e.g. dict[str, dict[str, list]].